src/server/courseInstance/__tests__/courseInstance.controller.test.ts

```diff
@@ -108,18 +108,11 @@ describe('Course Instance Controller', function () {
         });
       });
       context('With multiple valid acadYears', function () {
-        it('Should call the service once for each year passed', async function () {
+        it('Should only call the service for first year passed', async function () {
           const yearArgs = ['2018', '2020'];
           await ciController.getInstances(yearArgs.join(','));
-          strictEqual(getStub.callCount, yearArgs.length);
-        });
-        it('Should call the years in lexigraphical order', async function () {
-          const yearArgs = ['2020', '2019', '2018'];
-          await ciController.getInstances(yearArgs.join(','));
-          strictEqual(getStub.callCount, yearArgs.length);
-          strictEqual(getStub.args[0][0], 2018);
-          strictEqual(getStub.args[1][0], 2019);
-          strictEqual(getStub.args[2][0], 2020);
+          strictEqual(getStub.callCount, 1);
+          strictEqual(getStub.args[0][0], parseInt(yearArgs[0], 10));
```

Comment:
So the intended behavior is to call only the first year by index, with no consideration for sorting by chronology. So swapping the order of the `yearArgs` should sitll pass, since the assertion is comparing against the first element of the element. Does that make sense?

I made some changes to the test language in 36d245d3, and included an additional assertion the second arguement was not used. I also removed the test immediately after it, checking about the behavior with redundant years, since that doesn't really apply any more. Does that make more sense now?

Reply:
Ok that makes sense.